components/Topic.js

```diff
@@ -1,7 +1,22 @@
 import React from "react";
 
-const Topic = () => {
-	return <div>Topic</div>;
+const Topic = ({ Topic, BackgroundColor, TextColor }) => {
+	return (
+		<div className='flex w-full h-full'>
+			<div
+				className='self-center box-border h-24 w-48 border-2 border-black rounded-3xl py-3'
+				style={{
+					backgroundColor: BackgroundColor,
+					color: TextColor,
+					boxShadow: "8px -6px black",
+				}}
+			>
+				<div className='flex justify-center items-center py-3 font-lexend font-bold text-lg'>
+					{Topic}{" "}
```

Comment:
what is the `{" "}` for?

Reply:
oh when I run prettier it formats like this. Not sure why the "" is added.

Reply:
hmmm, look up the reason for that and put a rule in the prettier config file to fix that